src/ucp/core/ucp_listener.c

```diff
@@ -416,10 +434,18 @@ ucp_listen_on_iface(ucp_listener_h listener,
         goto err_close_listener_wifaces;
     }
 
-    listener->port = port;
+    status = ucs_sockaddr_copy((struct sockaddr *)&listener->sockaddr,
```

Comment:
i'm not sure we need the last part, where we make sure the uct listener actually listens on what we asked it to.
we don't have such checks in any other place AFAIR

Reply:
@yosefe which line numbers are you referring to?